openfisca-web-api/input-output-data.md

```diff
@@ -1,46 +1,25 @@
-# JSON data structures
+# Input and output data
 
-## axes
-
-A JSON axis is an object structured this way:
-* `count` (integer, >= 1, required): the number of steps to go from min to max
-* `index` (integer, >= 0, default: 0): the index of the person on which to apply the variation of the variable
-* `max` (integer or float, required): the maximum value of the varying variable
-* `min` (integer or float, required): the minimum value of the varying variable
-* `name` (string, one of available [variable names](https://legislation.openfisca.fr/variables), required): the name of the varying variable
-* `period` (a JSON period):
-
-### Parallel axes
-
-TODO
-
-### Perpendicular axes
-
-TODO
+## values by period
 
-## instants
+The value of a variable is always expressed in function of a definition period.
 
-TODO
-
-## parameters
+Example: `{salaire_de_base: {"2015-01": 5000}}`
 
-TODO
+When the period is not given, for example `enfant_a_charge: false`, the global period of the [scenario](#scenarios) is used.
 
-## periods
+Example: `{salaire_de_base: 5000}` is equivalent to `{salaire_de_base: {<scenario_period>: 5000}}`, `<scenario_period>` being defined in the [scenario](#scenarios).
 
-A JSON period can be:
-* a `string` like "2014", "2014-01", "2014-03:2"
-* an `object` like `{start: "2014", unit: "year"}`
+If a `set_input` attribute is defined in the simulation variable, it's possible to specify a different period unit and let the Core dispatch the value. See also: [periods section](../coding-the-legislation/35_periods.md).
```

Comment:
I hesitated adding this line and the following example. The fact that it is hard to understand confirms it's not a good idea. It's documented elsewhere so I remove it.